src/imageApiCalls.js

```diff
@@ -0,0 +1,13 @@
+import './Components/Inspirations/Inspirations.scss'
+
+export const getImages = () => {
+  return fetch('https://picsum.photos/v2/list')
```

Comment:
we should try to work with the 404 errors tomorrow

Reply:
sounds good, were you thinking of just doing different response statuses for different error codes? Any displaying of errors would need to be in the imagesData.tsx file.

Reply:
Agreed! I typically like to have separate pages for 404 and 500 errors to cover our buts for errors.